themes/default/content/docs/guides/self-hosted/components/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Pulumi Service Components
+menu:
+    userguides:
+        parent: self_hosted
+        identifier: self_hosted_components
+        weight: 70
+meta_desc: The Pulumi service container images page links and docker hub links.
+---
+
+{{% notes type="info" %}}
+The component repositories are private. [Contact us]({{< relref "/contact.md" >}}) if you would like to evaluate the Self-Hosted Enterprise Edition.
+{{% /notes %}}
+
+| Component | Repository |
+| --------- | ---------- |
+| [API]({{< relref "api" >}}) | [https://hub.docker.com/r/pulumi/service/](https://hub.docker.com/r/pulumi/service/) |
+| [Console]({{< relref "console" >}}) |	[https://hub.docker.com/r/pulumi/console/](https://hub.docker.com/r/pulumi/console/) |
```

Comment:
we aren't calling this the Pulumi Console or Console (capital C) anymore, but we can call this the Web console

```suggestion
| [Web console]({{< relref "console" >}}) |	[https://hub.docker.com/r/pulumi/console/](https://hub.docker.com/r/pulumi/console/) |
```